data/json/items/comestibles.json

```diff
@@ -13300,12 +13300,13 @@
         "type" : "COMESTIBLE",
         "id" : "apple_canned",
         "name" : "canned fruit",
+        "name_plural" : "canned fruit",
         "weight" : 450,
         "color" : "red",
         "addiction_type" : "none",
         "spoils_in" : 180,
         "stim" : 0,
-        "container" : "jar_glass",
+        "container" : "can_food",
```

Comment:
The description for just this item says, "Sealed glass jar containing preserved fruit.  Bland, mushy and losing color." 
 
All the rest are okay, though, I think.